api/routes/application/routes.py

```diff
@@ -226,7 +226,7 @@ def submit(self, application_id):
                 Notification.send(
                     notify_template,
                     account.email,
-                    full_name,
+                    full_name.capitalize(),
```

Comment:
can we use `.title()` here as .`capitalize()` only makes first letter word to uppercase